build-data-capturing-gradle-samples/capture-test-pts-support/gradle-test-pts-support.gradle.kts

```diff
@@ -86,3 +96,30 @@ class Capture(val logger: Logger) {
         }
     }
 }
+
+class KotestEngine(private val version: String) : Comparable<KotestEngine> {
+
+    constructor(jarFile: File) : this(jarFile.name.split("-")[jarFile.name.split("-").lastIndex].replace(".jar", ""))
```

Comment:
is the version always part of the jar file's name?

Reply:
It is